bc-react/src/stores/UserStore.js

```diff
@@ -36,6 +35,12 @@ class UserStore extends EventEmitter{
         this.emit('login-fail')
         break
       }
+      case("DELETE_USER"):{
```

Comment:
ok! do i delete it on atom after you merge or is there a way to do it from here?